slick-codegen/src/main/scala/slick/codegen/AbstractSourceCodeGenerator.scala

```diff
@@ -93,19 +129,24 @@ def $name($args): $name = {
 
     trait PlainSqlMapperDef extends super.PlainSqlMapperDef{
       def code = {
-        val positional = compoundValue(columnsPositional.map(c => (if(c.fakeNullable || c.model.nullable)s"<<?[${c.rawType}]"else s"<<[${c.rawType}]")))
+        def argList(values: Seq[String]) = values.mkString("(" ,", " , ")")
+        val positional = columnsPositional.map(c => (if(c.fakeNullable || c.model.nullable)s"<<?[${c.rawType}]"else s"<<[${c.rawType}]"))
         val dependencies = columns.map(_.exposedType).distinct.zipWithIndex.map{ case (t,i) => s"""e$i: GR[$t]"""}.mkString(", ")
-        val rearranged = compoundValue(desiredColumnOrder.map(i => if(hlistEnabled) s"r($i)" else tuple(i)))
-        def result(args: String) = if(mappingEnabled) s"$factory($args)" else args
+        val rearranged   =if( desiredColumnOrder.size <= maxTupleSize)
```

Comment:
I'd prefer if you'd stick to the whitespace-style we use almost everywhere. e.g. no `val rearranged   =if`